Sources/TuistDependencies/SwiftPackageManager/Utils/SwiftPackageManagerGraphGenerator.swift

```diff
@@ -86,6 +86,10 @@ public final class SwiftPackageManagerGraphGenerator: SwiftPackageManagerGraphGe
         let packageInfos: [
             (id: String, name: String, folder: AbsolutePath, targetToArtifactPaths: [String: AbsolutePath], info: PackageInfo)
         ]
+
+        // Platforms include iOS only
+        let shouldIgnoreMacCatalyst = platforms.subtracting([.iOS]).isEmpty
```

Comment:
I don't think this is the correct solution 🤔 
AFAIK maccatalyst is indeed for iOS targets. The change should rather be to enhance the Dependencies.swift to allow to decide whether maccatalyst should be enabled or not.

Given support for maccatalyst is already not working (because mac should be added also to the other dependencies, but we are not doing it), I would just change [this](https://github.com/tuist/tuist/blob/c4b543ce53c10851dcd174ed0fa7c78425caec1b/Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift#L557-L561) to ignore macCatalist 🤔

Reply:
Thanks for feedback 😀
It would be best to fix [that](https://github.com/tuist/tuist/blob/c4b543ce53c10851dcd174ed0fa7c78425caec1b/Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift#L557-L561) part.
I look forward to seeing what you change in the future.
I will use it by removing the macCatalyst support from library until it is updated.

Reply:
If someone were using macCatalyst, I thought they would include both iOS and macOS in Dependencies.platforms.

Reply:
macOS is to develop natively for macOS, macCatalyst is a layer on top of iOS that makes the iOS app runnable on macOS

Reply:
I have no experience with macCatalyst, so I was wrong. 😅